pom.xml

```diff
@@ -168,7 +168,10 @@ com.fasterxml.jackson.core.*;version=${project.version}
                 <filter>
                   <artifact>ch.randelshofer:fastdoubleparser</artifact>
                   <excludes>
+                    <exclude>META-INF/LICENSE</exclude>
+                    <exclude>META-INF/MANIFEST.MF</exclude>
                     <exclude>META-INF/versions/**/module-info.*</exclude>
+                    <exclude>META-INF/versions/22/**</exclude>
```

Comment:
Could you add an XML comment with reference to this issue (or one that triggered this), to explain why we are excluding JDK 22 implementation from Jackson 2.x.
For now I think 3.0 can retain newer versions.

Reply:
comment added

Reply:
it would probably be useful to port the LICENSE and MANIFEST.MF excludes to Jackson 3